src/wp-admin/includes/file.php

```diff
@@ -2795,7 +2795,7 @@ function wp_opcache_invalidate_directory( $dir ) {
 				__( '%s expects a non-empty string.' ),
 				'<code>wp_opcache_invalidate_directory()</code>'
 			);
-			trigger_error( $error_message );
+			wp_trigger_error( __FUNCTION__, $error_message );
```

Comment:
The function's name is already in the error message. It's redundant to add it again.
```suggestion
			wp_trigger_error( '', $error_message );
```